src/components/i18n/en.tsx

```diff
@@ -7,5 +9,14 @@ export const messages_en = {
   exit_editor: `Exit Editor`,
   home: `Home`,
   lists: `Lists`,
+  new_template_description:
+    "Have you finished editing your Template? Save it in the “My Templates” section to be able to use this version as many times as you want.",
+  new_template_has_been_saved:
+    "The Template was successfully saved in the “My Templates” section. The changes you make from now on, will not be applied.",
+  new_template_label: "Template Name:",
+  new_template_title: "Save as private Template",
   save: `Save`,
+  save_template: "Save as Template",
+  saving_changes:
+    "There has been an error saving changes, please try again later.",
```

Comment:
Are these the same texts than in MSEditor?

Reply:
Yes, is copy from MSEditor

Reply:
👍